__tests__/app.test.js

```diff
@@ -13,3 +21,24 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with all topics", () => {
+    return request(app)
+    .get("/api/topics")
+    .expect(200)
+    .then(({body})=>{
+      expect(body.topics.length).toBe(3);
```

Comment:
Nice inclusion of length here, it's worth me pointing out that there is a .toHaveLength() matcher that you can use also, both are fine.